example_project_structure/impls/lending/lending.rs

```diff
@@ -60,6 +60,13 @@ impl<T: LendingStorage<Data = LendingData> + PausableStorage<Data = PausableData
         Ok(PSP22Ref::total_supply(&mapped_asset))
     }
 
+    default fn get_shares_from_asset(&self, asset_address: AccountId) -> Result<AccountId, LendingError> {
```

Comment:
```suggestion
    default fn get_asset_shares(&self, asset_address: AccountId) -> Result<AccountId, LendingError> {
```

Not meant to be mean :smiley: but you want to get the share token of this asset not asset from the shares